.github/workflows/push_dist_to_npm.yml

```diff
@@ -0,0 +1,31 @@
+name: Push dist to npm
+on: push
```

Comment:
For now this runs on all branches, but once the integration with the GraphHopper server works we should probably limit this to the main branch